src/main/java/com/bullhornsdk/data/model/entity/core/standard/CorporateUser.java

```diff
@@ -159,6 +160,22 @@ public class CorporateUser extends CustomFieldsA implements QueryEntity, Associa
 
     private DateTime userDateAdded;
 
+    private Location addressSourceLocation;
```

Comment:
Do we need to add these new fields in @JsonPropertyOrder, getters, setters, etc?